assets/src/components/v2/full_line_map.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ComponentType, useState, useEffect } from "react";
+
+interface Props {
+  asset_urls: string[];
+}
+
+const intervalInMs = 10000
+
+const FullLineMap: ComponentType<Props> = ({asset_urls}) => {
+  const [assetIndex, setAssetIndex] = useState(0);
+
+  useEffect(() => {
+  if (assetIndex === asset_urls.length - 1) {
+    setTimeout(() => {setAssetIndex(0)}, intervalInMs)
+  } else {
+    setTimeout(() => {setAssetIndex((i) => i + 1)}, intervalInMs)
+  }
+}, [assetIndex])
```

Comment:
This will work for now because the list of asset URLs for a given screen is essentially unchanging, but in the future when the content starts being more real-time, we'll need to use the persistent carousel higher-order component instead of this `useEffect` to achieve the correct paging logic.